easybuild/easyconfigs/l/libelf/libelf-0.8.13-GCCcore-9.3.0.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'ConfigureMake'
+
+name = 'libelf'
+version = '0.8.13'
+
+homepage = 'http://www.mr511.de/software/english.html'
```

Comment:
@migueldiascosta wondering whether we can do anything about this homepage, which gives a 404 for me with http and a security error with https...

Reply:
We could use the page in the directory of `fsf.org`: https://directory.fsf.org/wiki/Libelf